scripts/bin/update-checksum

```diff
@@ -57,29 +57,35 @@ for package in "${@}"; do
 
 	# Calculate new checksum in subshell.
 	echo
-	new_checksum=$(
+	checksum=($(
 		source "${buildsh_path}" 2>/dev/null
+		echo ${TERMUX_PKG_SHA256[@]}
+	))
+	new_checksum=($(
+		source "${buildsh_path}" 2>/dev/null
+		for url in ${TERMUX_PKG_SRCURL[@]}; do
+			dl_tmpdir=$(mktemp -d "${TMPDIR-/tmp}/termux.src.dl.XXXXXXXX")
 
-		dl_tmpdir=$(mktemp -d "${TMPDIR-/tmp}/termux.src.dl.XXXXXXXX")
+			if [ -n "${dl_tmpdir}" ] && [ -d "${dl_tmpdir}" ]; then
+				if ! curl --fail --location --retry 3 --output "${dl_tmpdir}/source-bundle" \
+					"$url"; then
+					rm -rf "${dl_tmpdir}"
+				fi
 
-		if [ -n "${dl_tmpdir}" ] && [ -d "${dl_tmpdir}" ]; then
-			if ! curl --fail --location --retry 3 --output "${dl_tmpdir}/source-bundle" \
-				"${TERMUX_PKG_SRCURL[0]}"; then
-				rm -rf "${dl_tmpdir}"
-			fi
+				if [ -f "${dl_tmpdir}/source-bundle" ]; then
+					sha256sum "${dl_tmpdir}/source-bundle" | awk '{ print $1 }'
+				fi
 
-			if [ -f "${dl_tmpdir}/source-bundle" ]; then
-				sha256sum "${dl_tmpdir}/source-bundle" | awk '{ print $1 }'
+				rm -rf "${dl_tmpdir}"
 			fi
+		done
+	))
 
-			rm -rf "${dl_tmpdir}"
-		fi
-	)
-
-	if [ -n "${new_checksum}" ]; then
+	if [ -n "${checksum}" ] && [ -n "${new_checksum}" ]; then
 		# Replace old SHA-256.
-		# Note that this will not work properly if SHA-256 is an array.
-		sed -i "s/^\(TERMUX_PKG_SHA256=\)\(.*\)\$/\1${new_checksum}/g" "${buildsh_path}"
+		for ((i=0; i<${#new_checksum[@]}; ++i)) do
+			sed -i "s|${checksum[i]}|${new_checksum[i]}|" "${buildsh_path}"
```

Comment:
This breaks the old way of quickly populate empty or replace invalid `TERMUX_PKG_SHA256=` when prototyping `build.sh`